addons/crm/data/crm_stage_data.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo noupdate="1">
-    <record model="crm.stage" id="stage_lead1">
+    <record model="crm.stage" id="stage_lead1" forcecreate="False">
```

Comment:
Could you update test_01_crm_tour so that it does not depend on those data then ?

Reply:
Hello @tde-banana-odoo 
I updated the tour and made it independent from those data...
Can you please take a look at it.
Thanks!! :blush: